composer.json

```diff
@@ -22,7 +22,9 @@
     "zetacomponents/mail": "dev-1.7-civi",
     "phpoffice/phpword": "^0.13.0",
     "pear/Validate_Finance_CreditCard": "dev-master",
-    "civicrm/civicrm-cxn-rpc": "~0.16.12.05"
+    "civicrm/civicrm-cxn-rpc": "~0.16.12.05",
+    "pear/Net_SMTP": "1.6.*",
```

Comment:
hi @mickadoo allowing anything under 2.0 would be a problem as version 1.7 specifies as part of the composer.json that min PHP is 5.4 which > our min php of 5.3 and php 5.3 is what is installed on the test boxes